packages/link-bins/src/index.ts

```diff
@@ -122,7 +122,7 @@ async function getPackageBins (
     return []
   }
 
-  if (R.isEmpty(pkg.bin)) {
+  if (R.isEmpty(pkg.bin) && !target.includes('node_modules')) {
```

Comment:
Create a function and use it instead of `!target.includes('node_modules')`

```ts
async function isFromModules (filename: string) {
  const real = await fs.realpath(filename)
  return normalizePath(real).includes('/node_modules/')
}
```

The current implementation probably won't work because workspace packages are symlinked to node_modules, so you need to check the real location.